src/elements/content-uploader/ItemAction.js

```diff
@@ -54,7 +55,7 @@ const ItemAction = ({ status, onClick, intl, isFolder = false }: Props) => {
     return (
         <div className="bcu-item-action">
             <Tooltip position="top-left" text={tooltip}>
-                <PlainButton onClick={onClick} type="button" {...(retry ? { 'data-resin-target': 'uploadretry' } : {})}>
+                <PlainButton onClick={onClick} type="button" {...(target ? { 'data-resin-target': target } : {})}>
```

Comment:
To avoid the ternary, you could go with something like this:

```
let resin = {};

if (whatever) {
   resin.target = 'mytarget';
}

<PlainButton onClick={onClick} type="button" {...resin}>
```